src/ipc/prepare-boot-usb.ts

```diff
@@ -70,45 +54,25 @@ function parseEfiBootMgrOutput(output: string): BootOption[] {
 function prepareToBootFromUsb(): boolean {
   const { stdout: bootStdout } = exec('efibootmgr', ['-v']);
   const bootOptions = parseEfiBootMgrOutput(bootStdout);
-  const { stdout: lsblkStdout } = exec('lsblk', [
-    '-o',
-    'partuuid,name',
-    '-J',
-    '-l',
-  ]);
-  const rawData = JSON.parse(lsblkStdout) as RawDataReturn;
```

Comment:
Github is complaining about the import of RawDataReturn, take it out at the start of the file? (I'd leave a comment there but it won't let me...)

Reply:
good catch, fixed.